governance/dashboard-update.py

```
@@ -0,0 +1,292 @@
+        stats["created"] = 1 if created else 0
+        stats["issue"] = num
+        if num is None:  # dry-run 新建路径
+            print(f"AUDIT | butler=dashboard-update | trigger={TRIGGER} | outcome=ok | "
+                  f"dry-run=1 | actions={json.dumps(stats, ensure_ascii=False)}")
+            return 0
+        if not created:
+            cur = get(f"/repos/{ORG}/{HOME_REPO}/issues/{num}")
+            if (cur.get("body") or "").strip() == body.strip():
+                stats["unchanged"] = 1
+            elif DRY_RUN:
+                print(f"[dry-run] 将编辑 issue #{num} body（{len(body)} 字节）")
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)